external-builds/pytorch/pytorch_vision_repo.py

```diff
@@ -135,8 +132,34 @@ def add_common(command_parser: argparse.ArgumentParser):
     save_patches_p.set_defaults(func=repo_management.do_save_patches)
 
     args = p.parse_args(cl_args)
-    if args.require_related_commit and not HAS_RELATED_COMMIT:
-        raise ValueError("Could not find torchvision in pytorch/related_commits")
+    # Set default values based on the pin file in the pytorch repo.
+    (
+        default_git_origin,
+        default_git_hashtag,
+        default_patchset,
+        has_related_commit,
+    ) = repo_management.read_pytorch_rocm_pins(
+        args.torch_dir,
+        os="centos",  # Read pins for "centos" on Linux and Windows
+        project="torchvision",
+        default_origin=DEFAULT_ORIGIN,
+        default_hashtag=DEFAULT_HASHTAG,
+        default_patchset=DEFAULT_PATCHSET,
+    )
+
+    if args.require_related_commit and not has_related_commit:
+        raise ValueError(
+            f"Could not find torchvision in '{args.torch_dir}/related_commits' (did you mean to set a different --torch-dir?)"
+        )
+
+    # Priority order:
+    #   1. Explicitly set values
+    #   2. Values loaded from the pin in the torch repo
+    #   3. Fallback default values
+    args.gitrepo_origin = args.gitrepo_origin or default_git_origin
+    args.repo_hashtag = args.repo_hashtag or default_git_hashtag
+    args.patchset = args.patchset or default_patchset
```

Comment:
`repo_hashtag` still has a default set in argparse - is this on purpose? if yes then we should remove it here

Reply:
Can you take over and harmonize this across the scripts @HereThereBeDragons ?